src/util/antlr/ANTLRErrorHandling.cpp

```diff
@@ -2,38 +2,76 @@
 //  Chair of Algorithms and Data Structures.
 //  Author: Julian Mundhahs (mundhahj@informatik.uni-freiburg.de)
 
-#include <util/antlr/ANTLRErrorHandling.h>
+#include "ANTLRErrorHandling.h"
 
-#include "CharStream.h"
-#include "CommonTokenStream.h"
-#include "TokenSource.h"
+#include <Lexer.h>
+#include <Parser.h>
```

Comment:
Those are antlr-headers, right?
add a comment that they are antlr headers.

I am looking for a way to make this `antlr4/Lexer.h` etc...